.travis.yml

```diff
@@ -8,6 +8,12 @@ compiler:
   - gcc
   - clang
 
+env:
+  - CMAKE_FLAGS=
+  - CMAKE_FLAGS=-DTINYSPLINE_DOUBLE_PRECISION=NO
```

Comment:
By default, TinySpline is configured with single precision. So, shouldn't this be YES?

Reply:
Oh whoops, fixing that now!